website/docs/intro/install/docker.mdx

```diff
@@ -32,28 +32,21 @@ To pull platform-specific images (`amd64`, `arm`, `arm64`, `386`) use:
 
 ## Usage
 
-1. Create your `main.tf` file. Example:
-```hcl
-resource "null_resource" "default" {
-  provisioner "local-exec" {
-    command = "echo 'hello world!'"
-  }
-}
-```
-
-2. Pull the image from GitHub Packages registry:
-
-<CodeBlock language="bash">{DockerPullScript}</CodeBlock>
-
-3. Run OpenTofu as a Docker container:
-
-<CodeBlock language="bash">{DockerRunScript}</CodeBlock>
-
-:::tip
+To pull the image from GitHub Packages registry:
 
+:::note
 If you run into rate limiting issues with GitHub, create a
 [Personal Access Token](https://github.com/settings/tokens) and
 [log in](https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-container-registry#authenticating-to-the-container-registry)
 to the registry.
+:::
```

Comment:
I will include some screenshots to help us to decide on it. I personally prefer the following raking:

1. Above Option 2
2. Above Option 1
3. Down Option 2
4. Down Option 1

I only choose the option "2" because we have just 2 notes, but if we have more we can move to the option 1. What is your opinion on that?

Above:
Option 1: 
![image](https://github.com/opentofu/opentofu/assets/39157101/41fc8e85-7ed4-4a74-8825-3588fdcc3917)

Option 2:
![Screenshot from 2024-06-08 14-33-01](https://github.com/opentofu/opentofu/assets/39157101/d19c22e9-d903-4c23-b517-a8e5788702df)

Down:
Option 1:
![Screenshot from 2024-06-08 14-35-37](https://github.com/opentofu/opentofu/assets/39157101/fa11781f-2c40-4d31-8079-c8adb41f5109)

Option 2:
![Screenshot from 2024-06-08 14-34-23](https://github.com/opentofu/opentofu/assets/39157101/12761067-d1cc-42ec-bd01-a5b81e583e62)

Reply:
I like both option 1 and 2, with a slight preference for 2.  Let's go with option 2 for now and we can always refactor it if additional items need to be added in the future.